lab1/Makefile

```
@@ -0,0 +1,11 @@
+
+CC=g++
+# CFLAGS=-I.
+
+scytale: scytale.cpp
+	mkdir -p demo
+	$(CC) scytale.cpp simple_ciphers.cpp -o ./demo/scytale 
+
+ceasar: ceasar.cpp
```

Typo. His name is Julius Caesar :)

Please also fix that along the code.

```suggestion
caesar: caesar.cpp
```